plugin/completion/bin_complete.py

```diff
@@ -116,7 +116,7 @@ def exists_for_view(self, view_id):
         Returns:
             bool: compile flags exist for this view
         """
-        if view_id not in self.flags_dict:
+        if self.flags_dict.get(view_id) is None:
```

Comment:
It does make sense to me.

Checking with `buffer_id not in self.flags_dict` will return True if `self.flags_dict[buffer_id]` is `[]`. I'm assuming that is not really an expected behavior. But then I haven't looked much into the code to understand what's the expected behavior.